packages/toggleInputList/components/ToggleInputList.tsx

```diff
@@ -168,7 +133,7 @@ class ToggleInputList extends React.PureComponent<ToggleInputListProps, {}> {
         hintContent,
         id,
         inputLabel,
-        name: inputListId,
+        name: id,
```

Comment:
Could you explain a bit more on why this should be changed from `inputListId` which set name to `radio` as seen in the original snapshots to now it being set to an `id` value?

Reply:
Well noticed, that was my mistake :)

There were two conflicting `id` attributes and I put the wrong one at the `name` attribute.

I just fixed it in the last commit.